src/test/Makefile

```diff
@@ -592,7 +592,7 @@ GTEST_HEADERS = $(GTEST_DIR)/inc/gtest/*.h
 ## V                 : Set verbosity level based on the V= parameter
 ##                     V=0 Low
 ##                     V=1 High
-include ../../make/build_verbosity.mk
+include ../../mk/build_verbosity.mk
```

Comment:
(this should be referencing $(ROOT) anyway)

Reply:
Updated